src/components/ModalUnregisteredTokenInfo.js

```diff
@@ -62,7 +67,7 @@ class ModalUnregisteredTokenInfo extends React.Component {
       );
 
       try {
-        const tokenData = await hathorLib.tokensUtils.validateTokenToAddByConfigurationString(configurationString, this.props.wallet.storage);
```

Comment:
It seems this line was actually broken before your change :wrench:

Reply:
Yes, the wallet was not being passed as a prop